src/SignalR/common/Http.Connections/src/ConnectionsAppBuilderExtensions.cs

```diff
@@ -27,14 +24,13 @@ public static IApplicationBuilder UseConnections(this IApplicationBuilder app, A
                 throw new ArgumentNullException(nameof(configure));
             }
 
-            var dispatcher = app.ApplicationServices.GetRequiredService<HttpConnectionDispatcher>();
-
-            var routes = new RouteBuilder(app);
-
-            configure(new ConnectionsRouteBuilder(routes, dispatcher));
-
             app.UseWebSockets();
-            app.UseRouter(routes.Build());
+            app.UseRouting();
+            app.UseAuthorization();
+            app.UseEndpoints(endpoints =>
```

Comment:
@davidfowl 
Have any thoughts on this? Converting `app.UseSignalR(r => r.Map...())` to use Endpoints internally. This lets us get rid of our custom auth stuff in `HttpConnectionDispatcher`.

And side-effect, I believe will allow apps that port to 3.0 but don't change to endpoints to use the service automagically without `UseAzureSignalR`.

Reply:
This to me seems super spicy. Does it make sense to deprecate this API now that it's literally equivalent to something else? It's goodness if we can converge all of the code that we show in samples/docs.

Reply:
I would be ok deprecating the old stuff
@anurse @davidfowl ?

Reply:
I'm OK marking `UseSignalR` as obsolete if that's what we're talking about. We should have done that already anyway, since we want people to use `UseEndpoints`.

Reply:
I would be obsoleting:
`UseConnections`
`ConnectionsRouteBuilder`
`HubRouteBuilder`
`UseSignalR`

Reply:
Sounds good.